packages/security/src/components/StatusIndicator/StatusIndicator-story.js

```diff
@@ -63,6 +65,27 @@ const { statusIndicator: statusIndicatorProps, statusSteps: statusStepsProps } =
 const { basic, retry: retryProps } = statusIndicatorProps;
 
 storiesOf(patterns('StatusIndicator#legacy'), module)
+  .addDecorator((Story) => (
+    <>
+      <InlineNotification
+        className="page-layouts__banner"
+        actions={
+          <NotificationActionButton
```

Comment:
Since this one doesn’t include a new home, I think we can remove the `NotificationActionButton`.